src/utils/stitch.js

```diff
@@ -35,3 +38,15 @@ export const callStitchFunction = async (fnName, metadata, ...fnArgs) => {
         console.error(error);
     }
 };
+
+export const callStitchAuthenticationFunction = async (
```

Comment:
This was messy, there is another function right above this which is a wrapper for Stitch but needed some more work to get `metadata` and supporting different `appIds` sorted out, hence still WIP but would like to get feedback

Reply:
I'm wondering if we should separate the clients so we have one for content and one for APIs.

We could keep these separated so the logic on what each is doing is more clear.

Reply:
If by `client` you mean a different app on Stitch, then yes we have different apps (the `snooty` one handles content, the `devhub-authentication` one is new and will be for APIs).

Is this what you were referring to?

Reply:
Here I was just trying to re-use the Stitch-related code to support different `appId`s

Reply:
I meant more like `const contentClient` and `const apiClient` so we don't have to think about the logic between the two apps

Reply:
Oh I see, keeping them more intentionally separate, that makes some sense.

I'd like to try having a `stitch.js` with some common Stitch commands and then two modules which extend them, `snootyStitch.js` and `devhubApiStitch.js` to help solidify this distinction.